worlds/raft/Options.py

```diff
@@ -32,7 +32,13 @@ class FillerItemTypes(Choice):
     option_both = 2
 
 class IslandFrequencyLocations(Choice):
-    """Sets where frequencies for story islands are located."""
+    """Sets where frequencies for story islands are located.
+    Vanilla will keep frequencies in their vanilla, non-randomized locations.
+    Random on Island will randomize each frequency within its vanilla island, but will preserve island order.
+    Random Island Order will change the order you visit islands, but will preserve the vanilla location of each frequency unlock.
+    Random On Island Random Order will randomize the location containing the frequency on each island and randomize the order.
+    Progressive will randomize the frequencies to anywhere, but will always unlock the frequencies in vanilla order as the frequency items are received.
```

Comment:
```suggestion
    Progressive will randomize the frequencies to anywhere, but will always unlock the frequencies in the vanilla order as the frequency items are received.
```

Reply:
I think this change is unnecessary, as "vanilla order" is already referencing the frequencies. Adding "their" does not change this or make it any clearer that the frequencies are what are being referenced, nor that it's the vanilla order of the frequencies.

Reply:
Fair enough